src/network/protocol/messages/filteradd.zig

```diff
@@ -0,0 +1,78 @@
+const std = @import("std");
+const protocol = @import("../lib.zig");
+
+/// FilterAddMessage represents the "filteradd" message
+///
+/// https://developer.bitcoin.org/reference/p2p_networking.html#filteradd
+pub const FilterAddMessage = struct {
+    element: []const u8,
```

Comment:
you need a deinit method to free this

Reply:
@tdelabro I have a deinit method on line 59-61

`
    pub fn deinit(self: *Self, allocator: std.mem.Allocator) void {
        allocator.free(self.element);
    }
`